<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WorkersInformation</class>
 <widget class="QDialog" name="WorkersInformation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>417</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WorkersInformation</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="GroupBoxWorkers">
     <property name="title">
      <string>Worker information</string>
     </property>
     <layout class="QVBoxLayout" name="mainLayout">
      <item>
       <layout class="QGridLayout" name="propertiesGrid">
        <item row="7" column="0">
         <widget class="QLabel" name="p_lbl_PhoneNumber">
          <property name="text">
           <string>Phone</string>
          </property>
         </widget>
        </item>
        <item row="7" column="2">
         <widget class="QLineEdit" name="p_line_PhoneNumber"/>
        </item>
        <item row="5" column="2">
         <widget class="QLineEdit" name="p_line_FirstName"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="p_lbl_ID">
          <property name="text">
           <string>ID Worker</string>
          </property>
         </widget>
        </item>
        <item row="10" column="2">
         <widget class="QPushButton" name="p_button_AddPosition">
          <property name="text">
           <string>Add position</string>
          </property>
         </widget>
        </item>
        <item row="9" column="2">
         <widget class="QListWidget" name="p_list_Positions">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="p_lbl_FirstName">
          <property name="text">
           <string>First name</string>
          </property>
         </widget>
        </item>
        <item row="6" column="2">
         <widget class="QLineEdit" name="p_line_LastName"/>
        </item>
        <item row="0" column="2">
         <widget class="QLineEdit" name="p_line_ID">
          <property name="inputMask">
           <string/>
          </property>
          <property name="readOnly">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="8" column="2">
         <widget class="QLineEdit" name="p_line_Email"/>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="p_lbl_LastName">
          <property name="text">
           <string>Last name</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QLineEdit" name="p_line_Surname"/>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="p_lbl_Email">
          <property name="text">
           <string>Email</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="p_lbl_Surname">
          <property name="text">
           <string>Surname</string>
          </property>
         </widget>
        </item>
        <item row="11" column="2">
         <widget class="QPushButton" name="p_button_DeletePosition">
          <property name="text">
           <string>Delete position</string>
          </property>
         </widget>
        </item>
        <item row="9" column="0" rowspan="3">
         <widget class="QLabel" name="p_lbl_Positions">
          <property name="text">
           <string>Positions</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="p_layout_navigationButtons">
        <item>
         <widget class="QPushButton" name="p_button_Save">
          <property name="text">
           <string>Add worker</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>p_line_ID</tabstop>
  <tabstop>p_line_Surname</tabstop>
  <tabstop>p_line_FirstName</tabstop>
  <tabstop>p_line_LastName</tabstop>
  <tabstop>p_line_PhoneNumber</tabstop>
  <tabstop>p_line_Email</tabstop>
  <tabstop>p_list_Positions</tabstop>
  <tabstop>p_button_AddPosition</tabstop>
  <tabstop>p_button_DeletePosition</tabstop>
  <tabstop>p_button_Save</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
